step-templates/RavenDB - Smuggler - Moving Data between File Systems

```diff
@@ -0,0 +1,93 @@
+  },
+  "SensitiveProperties": {},
+  "Parameters": [
+    {
+      "Name": "ravenSmugglerPath",
+      "Label": "Raven Smuggler Path",
+      "HelpText": "Full path to the Smuggler EXE.\n\nFor example: **C:\\RavenDB\\Smuggler\\Raven.Smuggler.exe**",
+      "DefaultValue": "C:\\RavenDB\\Smuggler\\Raven.Smuggler.exe",
```

Comment:
Same question as on the previous file, perhaps better left blank. You've got the example in the help text, that should suffice.